ts/components/lightbox/LightboxGallery.tsx

```diff
@@ -23,23 +24,27 @@ export interface MediaItemType {
 
 type Props = {
   media: Array<MediaItemType>;
-  selectedIndex: number;
+  selectedIndex?: number;
+  onClose?: () => void;
 };
 
 export const LightboxGallery = (props: Props) => {
-  const { media } = props;
+  const { media, selectedIndex = -1, onClose } = props;
   const [currentIndex, setCurrentIndex] = useState(-1);
   const selectedConversation = useSelectedConversationKey();
 
   const dispatch = useDispatch();
 
   // just run once, when the component is mounted. It's to show the lightbox on the specified index at start.
   useEffect(() => {
-    setCurrentIndex(props.selectedIndex);
+    setCurrentIndex(selectedIndex);
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
```

Comment:
could this not me `const [currentIndex, setCurrentIndex] = useState(props.selectedIndex)` ?